src/Piral.Blazor.Tools/tasks/ManagePiletTask.cs

```diff
@@ -617,9 +617,9 @@ public override bool Execute()
 
                 if (canScaffold)
                 {
-                    var requireInstall = PreparePilet();
-
                     CopyConfigurationFiles();
+                    
+                    var requireInstall = PreparePilet();
```

Comment:
In general this cannot work as the prepare pilet has the initialization steps and is based on `NpmRegistry`, i.e., it might / will overwrite `.npmrc` files.

So either:

- Remove `.npmrc` support and only rely on `NpmRegistry`
- Remove `NpmRegistry` and only rely on `.npmrc`
- Define what takes precedence over what (maybe in case of `NpmRegistry` given an available `.npmrc` should yield a warning)

If we decide for (3) then `NpmRegistry` will always be used to pre-create an `.npmrc` file - so no handing over parameters to `pilet init`. As such nothing would be overwritten by swapping the two (and copy configuration files would rather be "initialize configuration files").

Reply:
I think 3rd idea would be the best for this use case. I'll start with implementation.

Reply:
Yeah in this case (going for 3) we'd keep the switch as proposed, but we need to create an .npmrc (if not available) in case of the `NpmRegistry` being not-null or empty.

So what we'd definitely need to avoid is to pass `NpmRegistry` on to `pilet init`; instead we always want the copied / created .npmrc to be used.

Thanks for your efforts!